### PR TITLE
Adjustments for balance and difficulty scaling with more players

### PR DESCRIPTION
**Pull request by FPaizSF**
Fewer imps are placed by default in the level by default
Imp spawner added to first room
Imp spawners create imps with increased aggro distances
Imp spawners in boss room create imps that will aggro players in the middle of the room
Player main attack and hitpoints tuned
Rogue description of main attack now matches art and is slower but more powerful than tank's hammer

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.coop?pullRequest=321) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.coop?pullRequest=321) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: jdepuy<br/>:white_check_mark: eheimburg<br/>:x: FPaizSF<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.samples.coop?pullRequest=321) it.</sub>

File: Assets/BossRoom/Prefabs/Game/StaticNetworkObjects.prefab [hunk 1]
```diff
@@ -39,6 +39,9 @@ Transform:
   - {fileID: 3359597238691122721}
```
Comment (fernando-cortez) [inline Assets/BossRoom/Prefabs/Game/StaticNetworkObjects.prefab:39]: Not for this file in particular but a comment on BossRoom scene.

It seems the BossRoom scene didn't need to be marked as changed if only the `Dungeon` and `StaticNetworkObjects` prefabs were modified. Requesting that the BossRoom scene be reverted. Followed up offline with more details.

Comment (eheimburg) [reply Assets/BossRoom/Prefabs/Game/StaticNetworkObjects.prefab:39]: The scene had scene-specific overrides to the Dungeon prefab, so the scene had to be modified to lock in the changes and let the prefabs be authoritative.

Comment (eheimburg) [reply Assets/BossRoom/Prefabs/Game/StaticNetworkObjects.prefab:39]: Actually, the scene still had overrides in it, in this case for `StaticNetworkObjects`. I just saved those changes to the prefab, and checked in the scene again. So now the scene should have no overrides for the major prefabs, which will let changes happen to just the prefabs going forward. (Barring anything that needs navmesh regen, at least.)

Comment (fernando-cortez) [reply Assets/BossRoom/Prefabs/Game/StaticNetworkObjects.prefab:39]: Great. Thanks for the clarification.

**Review by fernando-cortez — APPROVED** [review]

**Review by dwoodruffsf — APPROVED** [review]

**Pull request merged**